src/components/explore/SourcesBreadCrumbs.tsx

```diff
@@ -13,16 +14,17 @@ export const SourcesBreadCrumbs = ({ slugPaths, current }: { slugPaths: string[]
   const pathnameArray = pathname.replace(`/${language}`, "").split("/");
   const isNotSourcesPage = navListWithoutSources.includes(pathnameArray[1]);
 
+  const isEnglishSlug = language === "en" && language.length == 2 && !LanguageCodes.includes(language);
```

Comment:
```suggestion
  const isEnglishSlug = language === "en"
```
the first check should suffice, if "en" then language.length is already 2, and checking if not in LanguageCodes doesn't necessarily imply that it is english language